src/dialogs/quick-bar/ha-quick-bar.ts

```diff
@@ -279,24 +301,44 @@ export class QuickBar extends LitElement {
     }
   }
 
-  private _generateCommandItems(): QuickBarItem[] {
-    return [
-      ...this._generateReloadCommands(),
-      ...this._generateServerControlCommands(),
-    ].sort((a, b) => compare(a.text.toLowerCase(), b.text.toLowerCase()));
+  private _generateEntityItemsIfEmpty(): QuickBarItem[] {
+    return this._entityItems
+      ? this._entityItems
+      : Object.keys(this.hass.states)
+          .map((entityId) => ({
+            text: computeStateName(this.hass.states[entityId]),
+            altText: entityId,
+            icon: domainIcon(
+              computeDomain(entityId),
+              this.hass.states[entityId]
+            ),
+            action: () => fireEvent(this, "hass-more-info", { entityId }),
+          }))
+          .sort((a, b) => compare(a.text.toLowerCase(), b.text.toLowerCase()));
+  }
+
+  private _generateCommandItemsIfEmpty(): QuickBarItem[] {
+    return this._commandItems
+      ? this._commandItems
```

Comment:
I rather have this function just generate the items and have the check in `_initializeItemsIfNeeded`

Reply:
done